### DISCUSSION TITLE
Ui view console terminal

### DISCUSSION THREAD

**Discussion by Angelk90** [zed-industries/zed#22087] [Feature Requests]
As you can see from the image in the zed layout the tabs at the top are opened, if you open so many terminals it is confusing and it is not even possible to rename the tabs.

In the style of vs code this way it is more tidy.

**Please vote and let us know your thoughts on this.**

Vscode:
<img width="550" alt="Screenshot 2024-05-26 alle 10 18 23" src="https://github.com/zed-industries/zed/assets/20476002/fa8dc853-2986-41ec-86a1-5836f6abcaab">

Zed:
<img width="643" alt="Screenshot 2024-05-26 alle 10 21 06" src="https://github.com/zed-industries/zed/assets/20476002/5a4ad624-ce44-4125-a871-1e61f959a956">

Suggestions:
<img width="504" alt="Screenshot 2024-05-26 alle 10 31 51" src="https://github.com/zed-industries/zed/assets/20476002/f2a8074f-2687-43a9-80b3-31a11581f675">

![image](https://github.com/zed-industries/zed/assets/20476002/941ca338-0ea9-4fbb-815e-7e95e123eb22)

https://user-images.githubusercontent.com/806104/161374050-7e8cc0b5-e731-49a3-a7de-6fd41328adea.mp4

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
